pkgs/development/python-modules/myfitnesspal/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchPypi, buildPythonPackage
+, blessed, keyring, keyrings-alt, lxml, measurement, python-dateutil, requests, six
+, mock, nose }:
+
+buildPythonPackage rec {
+  pname = "myfitnesspal";
+  version = "1.13.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "4f785341f0499bad8d3400cfcfffd99b7fcf8aac3971390f8ec3dfaed8361b20";
+  };
+
+  # Remove overly restrictive version constraints on keyring and keyrings.alt
+  postPatch = ''
+    sed -i 's/keyring>=.*/keyring/' requirements.txt
+    sed -i 's/keyrings.alt>=.*/keyrings.alt/' requirements.txt
+  '';
```

Comment:
Both modules pass all of their test cases, but I also verified manually that the package works; keyring appears to increment major versions very frequently so I just stripped the version bound entirely rather than loosening it to what we currently have.